Automattic/newspack-blocks
# feat(homepage-articles): rename block and reorganise settings

### All Submissions:






### Changes proposed in this Pull Request:

You can read more about the logic behind this change here: P2-pfaSwu-iT

The main reasons are:

- The block's functionality extends beyond just Posts for the Homepage
- The sidebar settings are disorganised, making navigation through options difficult

__Before:__

![homepage-posts-settings-before](https://github.com/user-attachments/assets/ba734fe7-abf2-49ac-a2aa-9f2a7f763e7f)

__After:__

![content-loop-settings-after](https://github.com/user-attachments/assets/39fb7e89-088d-4dd1-a74d-6b215ad390e7)
![content-loop-styles](https://github.com/user-attachments/assets/e195db0a-6c0a-46fb-9047-44087459e840)

### How to test the changes in this Pull Request:

1. Add a bunch of Homepage post block to a page with various settings
2. Switch to this branch
3. Check if the blocks are still displaying the correct information and has kept the correct settings

### Other information:
